LICENSE.md

```diff
@@ -0,0 +1,46 @@
+# License for branding assets
+
+Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)
+Copyright (C) ?-2021 The Devicon team
```

Comment:
I suggest to remove the date, you don't want to update this every year ;-)

```suggestion
Copyright (C) The Devicon team
```

Reply:
I think this is actually necessary. 😅